assets/animations.js

```diff
@@ -0,0 +1,25 @@
+function onIntersection(elements, observer) {
+  elements.forEach((element, index) => {
+    if (element.isIntersecting) {
+      const elementTarget = element.target;
+      elementTarget.classList.add('animation__trigger--active');
```

Comment:
Let's move `'animation__trigger--active'` and other such class names into a constant at the top of the file.

Reply:
I know this is a pattern in some codebases, but I haven't seen it in Dawn. And since we're only using the class in one place, I don't see the benefit of it.

Reply:
There are two big benefits that I see here:
1) Let's you write self-documenting code. Instead of `elementTarget.classList.add('animation__trigger--active')` you'll have `elementTarget.classList.add(SCROLLED_INTO_VIEW_CLASSNAME)` which is more easily understandable.
2) It avoids nesting magic words and numbers deep in the code. Wherever we are passing configuration values, classnames, numbers, etc. we should really store them in variables at the top of classes/modules to avoid hunting around for them or guessing what the values may be.